<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="logText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>411</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ColorMinLabel">
        <property name="text">
         <string>ColorMin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ColorMinChannelOption"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ColorMinValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ColorMinChannelSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="ColorMaxLabel">
        <property name="text">
         <string>ColorMax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ColorMaxChannelOption"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="ColorMaxValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ColorMaxChannelSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="DefineColorLabel">
        <property name="text">
         <string>DefinedColor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DefinedColorOption"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addTotrackingButtom">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Color to Tracking</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentSegmentationImg">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="trackedColorsList">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButtom">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="InputImg">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="OutputImg">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
